pkg/platform/runtime/envdef/environment.go

```diff
@@ -405,6 +405,25 @@ func (ed *EnvironmentDefinition) ExecutablePaths() (ExecutablePaths, error) {
 	return exes, nil
 }
 
+func (ed *EnvironmentDefinition) ExecutableDirs() (ExecutablePaths, error) {
+	exes, err := ed.ExecutablePaths()
+	if err != nil {
+		return nil, errs.Wrap(err, "Could not get executable paths")
+	}
+	fmt.Println("Exepaths:", exes)
+
+	var dirs ExecutablePaths
+	for _, p := range exes {
+		dir := filepath.Dir(p)
+		if funk.Contains(dirs, dir) {
+			continue
+		}
```

Comment:
Let's instead do a `funk.Uniq()` at the end. That'd make only one additional loop rather than having it loop the same entries inside another loop.